riva/proto/riva_nmt.proto

```diff
@@ -76,8 +76,12 @@ message SynthesizeSpeechConfig {
 /*
 *
 */
+
 message StreamingTranslateSpeechToSpeechResponse {
+    oneof streaming_response {
         nvidia.riva.tts.SynthesizeSpeechResponse speech = 1; //from riva_tts.proto
+        bool is_final=2; //end of tts stream
+    }
```

Comment:
Fix spacing.

Reply:
agree, we should add a comment for the StreamingTranslateSpeechToSpeechResponse message itself and mention one line detail for this flag as well.

Reply:
done.